blog/docs/events/knative-at-kubecon-na-2022.md

```diff
@@ -0,0 +1,49 @@
+# Knative at KubeCon ’22 North America
+
+**Date: 2022-10-20**
+
+There's less than a week until
+[KubeCon + KnativeCon Europe 202](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)
+kicks off in Detroit, MI.
+We are excited that the Knative community will be well represented at both events,
+and that once again we are hosting a full day of Knative content at the
+[KnativeCon](https://events.linuxfoundation.org/knativecon-north-america/) co-located event.
+
+We wanted to share with you a few of the Knative
+talks you can attend at the conferences to learn more about the Knative project and
+its progress in the past year.
+
+## Monday, October 24 KnativeCon
+
+Join us for a full day of Knative content at KnativeCon. This year we have a
+variety of talks covering Knative's core components, use cases, and the future of
+the project. Check out the full schedule on [Sched](https://knativeconna22.sched.com/).
```

Comment:
I feel like it would help to include times for these. You can either use a `<table>` tags, or a separate bolded header line.

e.g.

```md
## Monday, October 24: KnativeCon

**All Day**

Join us for ...
```

You might want to link the schedule / co-located event details in the title, and include how to add this to your registration in the body.